inductiva/fluids/wind_tunnel/wind_tunnel.py

```diff
@@ -74,25 +73,25 @@ def __init__(self,
         """
         if flow_velocity is None:
             logging.info("Using a default flow velocity: [30, 0, 0].")
-            self.flow_velocity = [30, 0, 0]
+            self.params["flow_velocity"] = [30, 0, 0]
         elif len(flow_velocity) != 3:
             raise ValueError("`flow_velocity` must have 3 values.")
         elif np.linalg.norm(np.array(flow_velocity)) > 100:
             raise ValueError("The `flow_velocity` magnitude is too high,"
                              " it must be less than 100 m/s.")
         else:
-            self.flow_velocity = flow_velocity
+            self.params["flow_velocity"] = flow_velocity
 
         if domain is None:
```

Comment:
I just noticed it, but the code will look cleaner if you provide this value as default in the template itself, you can look it up [here](https://support.sendwithus.com/jinja/default/#:~:text=Jinja%20has%20a%20built%20in,when%20one%20isn%27t%20provided.&text=If%20the%20value%20isn%27t,value%20is%20defined%20but%20blank.)

Reply:
I agree with you about being more clean, however it will be a bit more obscure for devs looking in, since they will have to check multiple files. However, it is a nice thing to explain for users! Thanks!!